src/components/Book.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+import { RemoveBookButton } from './Button';
+import '../styles/book.css';
+import progress from '../Assets/progress.png';
+
+const Book = ({
+  id, title, author, category,
+}) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveClick = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <div className="book">
+      <div className="book-cover">
+        <div className="book-info">
+          <h6 className="category">
+            {category}
+            Action
+          </h6>
+          <h3 className="title">
+            {title}
+          </h3>
+          <p className="author">
+            {author}
+          </p>
+        </div>
+        <div className="book-actions">
+          <p className="comments">
+            Comments
+          </p>
+          <span className="remove">
+            <RemoveBookButton className="remove-btn" type="button" onClick={handleRemoveClick}>Remove</RemoveBookButton>
+          </span>
+          <p className="edit">
+            Edit
+          </p>
+        </div>
+      </div>
+      <div className="progress">
+        <img src={progress} alt="progress" />
+        <div className="progress-info">
+          <p className="parcent">64%</p>
+          <p className="completed">Completed</p>
+        </div>
+      </div>
+      <div className="chapter">
+        <p className="current">CURRENT CHAPTER</p>
+        <p className="chapter-number">Chapter 17</p>
+        <button type="button" className="update-progress"><p>UPDATE PROGRESS</p></button>
+      </div>
+    </div>
```

Comment:
- [ ] Please kindly ensure to design this section in accordance to the given template. I observed that the `UPDATE PROGRESS BUTTON` is not properly place in its parent container and has a bold black outline. 💁‍♂️ 
- [ ] CURRENT CHAPTER should be displayed on a line. ℹ️ 
- [ ] The user icon is to far from the other components, maybe because there is an horizontal overflow. 👍 

## Your Design
<img width="960" alt="5" src="https://github.com/samuelkinyanjui/bookstoreredux/assets/103386183/e07110ca-c24f-41d8-ba6b-c016f0eae9c1">

## Given Template
<img width="474" alt="6" src="https://github.com/samuelkinyanjui/bookstoreredux/assets/103386183/05cdcf40-ab79-4e72-a9f7-77430851dc46">